### DISCUSSION TITLE
How to remove an IP from the Blocklist ?

### DISCUSSION THREAD

**Discussion by Thiteago** [kickstarter/rack-attack#613] [Questions (Q&A)]
I'm trying to remove an ip from the blocklist, but i can't find anyway to do that on the documentation, i'm using the default Rails cache if it's help someway

**Comment by courtenay** [comment]
Old discussion, but.. how we do it is we accessing the cache keys directly from an admin interface.  See the docs [here](https://github.com/rack/rack-attack/blob/0fbc9a1c460dc83f9046c6c908e7096201da0c43/docs/advanced_configuration.md?plain=1#L70)
